build-system/tasks/bundle-size.js

```diff
@@ -21,7 +21,7 @@ const gulp = require('gulp-help')(require('gulp'));
 const log = require('fancy-log');
 
 const runtimeFile = './dist/v0.js';
-const maxSize = '77.15KB';
+const maxSize = '77.21KB';
```

Comment:
The very next build on `master` went past this limit :(

https://travis-ci.org/ampproject/amphtml/jobs/375136775#L641

Looks like we need to provide a larger buffer :/